object-cache.php

```diff
@@ -1524,6 +1531,10 @@ public function getAllOptions() {
 			return array();
 		}
 
+		// getMulti returns a map of `[ cache_key => value ]` but we need to
+		// return a map of `[ option_name => value ]`
+		$data = array_combine( array_keys( $keys ), array_values( $data ) );
```

Comment:
We should put `array_keys()` on the `$this->get` call to avoid doing it twice. Also, don't _think_ `array_values` is needed.